src/middleware.js

```diff
@@ -1,12 +1,9 @@
 import Problem from './index'
 
-const CONTENT_TYPE = 'application/problem+json'
-
 export default function Middleware (space = null) {
   return (err, req, res, next) => {
     if (err instanceof Problem) {
-      res.set('Content-Type', CONTENT_TYPE)
-      res.send(JSON.stringify(err, null, space))
+      err.send(res, space)
```

Comment:
this would only work in Express, turning this into an Express specific Middleware, rather than a global one ...

also, setting the content type is important.